core/src/main/java/com/linecorp/armeria/client/routing/endpoint/healthcheck/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * {@link com.linecorp.armeria.client.routing.endpoint.healthcheck.HealthCheckedEndpointGroup} groups.
+ */
+package com.linecorp.armeria.client.routing.endpoint.healthcheck;
```

Comment:
Not directly related with this PR, but it seems to me that package hierarchy is getting too deep. Let me send a follow-up PR that simplifies this.